### DISCUSSION TITLE
Please add Maintainerr for Plex

### DISCUSSION THREAD

**Discussion by StaRipper** [tteck/Proxmox#3552] [Ideas]
Please add Maintainerr for Plex

https://github.com/jorenn92/Maintainerr

**Comment by StaRipper** [comment]
Installation:

https://docs.maintainerr.info/Installation/

**Comment by ghost** [comment]
Would love to see this added too! In desperate need of something to manage my library

**Comment by oOStroudyOo** [comment]
I guess for this you'd make it a stand-alone script, and have it as an optional install with the the Plex Media Server script.
